integration-tests/src/test/java/oracle/kubernetes/operator/ItOperatorUpgrade.java

```diff
@@ -293,6 +229,20 @@ private void checkDomainRollingRestarted() throws Exception {
     }
   }
 
+  /**
+   * Check whether the weblogic server instances are still RUNNING 
+   * not restarted due to Operator Upgrade
+   *
+   * @throws Exception If restarted
+   */
+  private void checkDomainNotRestarted() throws Exception {
+    TestUtils.checkPodReady(DUID + "-" + domain.getAdminServerName(), DOM_NS);
+    for (int i = 2; i >= 1; i--) {
+      LoggerHelper.getLocal().log(Level.INFO,
+          "Checking if managed server pod(" + DUID + "--managed-server" + i + ") is RUNNING");
+      TestUtils.checkPodReady(DUID + "-managed-server" + i, DOM_NS);
```

Comment:
the comment says we want to check they have not been restarted, but this just checks if they are running, not if they wer restarted.  is the comment wrong? or does the code need to be improved to check for restart?

Reply:
I added the logic to check the creationTimesamp before and after upgrade to make sure that the server pod has not been re-stared due to upgrade.